### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Malvoz** [W3CGHtmail/html-email-spec#3]
Supposedly there'd be accessibility recommendations for clients/authors. WCAG is applicable of course, but there may be limitations and interoperability issues due to lack of support for HTML, CSS and ARIA.

What are common accessibility issues in emails? And how do/should clients and/or authors deal with them?

Resource with related links by @wilbertheinen:
https://github.com/wilbertheinen/accessible-email-documentation/blob/master/README.md
